examples/06-verif-manual/vm-006-pinched_cylinder.py

```diff
@@ -0,0 +1,520 @@
+# Define Boundary Conditions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Application of symmetric boundary conditions for simplified model.
+
+# Select nodes by location and apply BC.
+mapdl.nsel("S", "LOC", "X", 0)
+mapdl.dsym("SYMM", "X", 0)
+mapdl.nsel("S", "LOC", "Y", 0)
+mapdl.dsym("SYMM", "Y", 0)
+mapdl.nsel("S", "LOC", "Z", 0)
+mapdl.dsym("SYMM", "Z", 0)
+mapdl.nsel("ALL")
+
+
+###############################################################################
+# Define Distributed Loads
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Apply the force of :math:`F = (100/4) lb` in the y-direction.
+
+# Parametrization of the :math:`F` load for the quarter of the model.
+F = 100 / 4
+
+# Application of the load to the model.
+mapdl.fk(top_keypoint, "FY", -F)
+mapdl.finish()
+
+
+###############################################################################
+# Solve
+# ~~~~~
+# Enter solution mode and solve the system. Print the solver output.
+
+mapdl.run("/SOLU")
+out = mapdl.solve()
+mapdl.finish()
+print(out)
+
+
+###############################################################################
+# Post-processing
+# ~~~~~~~~~~~~~~~
+# Enter post-processing for the model with elements ``shell181``.
+# Plotting nodal displacement.
+# Get the the radial displacement at the node where force F is applied.
+
+mapdl.post1()
+mapdl.set(1)
+
+
+###############################################################################
+# Plotting
+# ~~~~~~~~
+# Plot nodal displacement with PyVista functionality.
+mapdl.post_processing.plot_nodal_displacement(
+    title="Nodal Displacements",
+    component="Y",
+    cpos="zx",
+    scalar_bar_args={"title": "Nodal Displacements", "vertical": True},
+    show_node_numbering=True,
+    show_axes=True,
+    show_edges=True
+)
+
+
+###############################################################################
+# Getting the radial displacements
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# To determine the radial displacement :math:`\delta` at the point
+# where F is applied, we can use :meth:`Mapdl.get_value <ansys.mapdl.core.Mapdl.get_value>`.
+
+# Select keypoint by its number ``top_keypoint``.
+mapdl.ksel("S", vmin="top_keypoint")
+
+# Select the node associated with the selected keypoint.
+mapdl.nslk()
+
+# Get the number of the selected node by :meth:`Mapdl.get <ansys.mapdl.core.Mapdl.get>`
+top_node = int(mapdl.get("_", "node", 0, "num", "max"))
+
+# Check the number of the selected node.
+print(top_node)
+
+# Define radial displacement at the node where F is applied.
+deflect_shell_181 = mapdl.get_value(entity='node',
+                                    entnum=top_node,
+                                    item1='u',
+                                    it1num='y')
+
+# Use ``round()`` function to obtain needed number of decimals.
+print(round(deflect_shell_181, 4))
+
+
+###############################################################################
+# Redo with Shell 281
+# ~~~~~~~~~~~~~~~~~~~
```

Comment:
This second section is a duplication of the previous section. I'm fine with the duplication for the example, but we may consider using python functions for pre-processing so we don't have to duplicate the setup functions.

This is up to you. If you choose to duplicate it here, please apply the above sections to this section of the example.